harness/determined/pytorch/deepspeed/_deepspeed_context.py

```diff
@@ -65,15 +66,57 @@ class DeepSpeedTrialContext(det.TrialContext, pytorch._PyTorchReducerContext):
     5. Disable automatic gradient aggregation for non-pipeline-parallel training.
     """
 
-    def __init__(self, *args: Any, **kwargs: Any) -> None:
-        det.TrialContext.__init__(self, *args, **kwargs)
+    def __init__(
+        self,
+        core_context: det.core.Context,
+        trial_seed: Optional[int],
+        hparams: Optional[Dict],
+        slots_per_trial: int,
+        num_gpus: int,
+        exp_conf: Optional[Dict[str, Any]],
+        aggregation_frequency: int,
+        steps_completed: int,
+        debug_enabled: bool,
```

Comment:
this isn't getting used?

Reply:
pls make a pass and look for params that aren't being used, these will be user-facing APIs, and if we decide to get rid of them later they'll be breaking changes.